pubspec.yaml

```diff
@@ -9,8 +9,8 @@ topics:
   - cli
 
 environment:
-  sdk: ^3.0.0
+  sdk: ^3.3.0
```

Comment:
Do we really need as late as 3.3?

Reply:
In my mind: it hurts NOTHING. Makes sure we're getting access to latest language features. Given how quickly our ecosystem moves forward, etc.